src/components/Footer/Footer.jsx

```diff
@@ -34,8 +34,9 @@ export default function Footer() {
                 </button>
               </div>
               <p className={styles.agreementMessage}>
-                Натискаючи кнопку «Підписатись», ви даєте згоду на обробку
-                персональних даних
+                Натискаючи кнопку «Підписатись», ви даєте згоду
+                <br/>
+                на обробку персональних даних
```

Comment:
this is a smelly hack , needs to be styled differently
in your browser open dev tools and check the boundaries of elements:
- div.logoCotainer
- form.subscribeForm
- div.inputGroup
- and then hover over this p level - it just takes width of its parent - form.

why div.inputGroup doesn't take width of form ? 

<img width="841" height="625" alt="image" src="https://github.com/user-attachments/assets/ce0fc389-bcb5-4581-9d7e-595e5be06ff7" />